synthesizer/src/vm/verify.rs

```diff
@@ -243,9 +243,11 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                             .block_store()
                             .find_block_height_from_state_root(execution.global_state_root())?
                             .unwrap_or_default();
-                        let (cost, (_, _)) = match block_height < N::CONSENSUS_V2_HEIGHT {
-                            true => execution_cost_v1(&self.process().read(), execution)?,
-                            false => execution_cost_v2(&self.process().read(), execution)?,
+                        let (cost, (_, _)) = match block_height {
```

Comment:
If we had a utility like `N::CONSENSUS_VERSION(height: u32) -> ConsensusVersion`, this block could be rewritten as: 
```
match N::CONSENSUS_VERSION(block_height) {
    ConsensusVersion::V1 => execution_cost_v1(&self.process().read(), execution)?
    _ => execution_cost_v2(&self.process().read(), execution)?
}
```
Would that improve ergonomics? Ditto for similar patterns in this PR.

Reply:
Great idea 🙏 9d26eadfa